mix.exs

```diff
@@ -0,0 +1,53 @@
+defmodule TechForGoodNearYou.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :tech_for_good_near_you,
+     version: "0.0.1",
+     elixir: "~> 1.4",
+     elixirc_paths: elixirc_paths(Mix.env),
+     compilers: [:phoenix, :gettext] ++ Mix.compilers,
+     start_permanent: Mix.env == :prod,
+     aliases: aliases(),
+     deps: deps()]
+  end
+
+  # Configuration for the OTP application.
+  #
+  # Type `mix help compile.app` for more information.
+  def application do
+    [mod: {TechForGoodNearYou.Application, []},
+     extra_applications: [:logger, :runtime_tools, :httpotion]]
+  end
+
+  # Specifies which paths to compile per environment.
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_),     do: ["lib"]
+
+  # Specifies your project dependencies.
+  #
+  # Type `mix help deps` for examples and options.
+  defp deps do
+    [{:phoenix, "~> 1.3.0-rc"},
+     {:phoenix_pubsub, "~> 1.0"},
+     {:phoenix_ecto, "~> 3.2"},
+     {:postgrex, ">= 0.0.0"},
+     {:phoenix_html, "~> 2.6"},
+     {:phoenix_live_reload, "~> 1.0", only: :dev},
+     {:gettext, "~> 0.11"},
+     {:cowboy, "~> 1.0"},
+     {:httpotion, "~> 3.0.2"}]
```

Comment:
They're effectively the same thing, but Poision seems to have better https support, also given that Phoenix is so convention driven it's probably best to stick to the consensus unless we really need Potion for some reason